curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/63e9e5bcec5d33d21b82397e.md

```diff
@@ -0,0 +1,160 @@
+---
+id: 63e9e5bcec5d33d21b82397e
+title: Step 3
+challengeType: 0
+dashedName: step-3
+---
+
+# --description--
+
+A good way to test that everything is working is to log the `value` attribute of `numberInput` to the console. As a reminder, you can access the `value` attribute of an element by using dot or bracket notation.
+
+Within the `checkUserInput` function, use `console.log()` to log the `value` of `numberInput` to the console.
+
+# --hints--
+
+Your `console.log()` call should be inside the `checkUserInput` function.
+
+```js
+assert.match(String(checkUserInput), /console\.log\(\s*numberInput/);
```

Comment:
This test should just check if they added a console statement.
Because if I have this code here inside the correct function, it still fails even though I technically did it correctly.
This could be misleading to campers. 
<img width="731" alt="Screenshot 2023-12-12 at 3 07 50 PM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/67210629/2be37e0a-c0cd-47ed-8b9b-66b3ecdccb30">



This update works though



```suggestion
assert.match(String(checkUserInput), /console\.log\(\s*.*/);
```